internal/state/indexer/block/kv/kv.go

```diff
@@ -65,7 +65,7 @@ func (idx *BlockerIndexer) Index(bh types.EventDataNewBlockHeader) error {
 	}
 
 	// 2. index FinalizeBlock events
-	if err := idx.indexEvents(batch, bh.ResultFinalizeBlock.Events, types.EventTypeFinalizeBlock, height); err != nil {
+	if err := idx.indexEvents(batch, bh.ResultFinalizeBlock.Events, "finalize_block", height); err != nil {
```

Comment:
Do we use `indexEvents` for anything other than the "finalize_block" type? Just trying to figure out what it would look like to remove it entirely

Reply:
We only index event metadata for the block itself (these) and the transactions in the block, so as far as I can tell this is the only use. 

The question is whether it affects the query logic to remove the type suffix. I think probably not—it appears to only search for a prefix not including this tag. But the search code is a little complex so I didn't want to fold that into this change.